### PR TITLE
Feature/1.6/build time visibility

### PR DESCRIPTION
**Pull request by lazan**

### CODE REVIEW

File: middleware/configuration/documentation/domain.general.operation.md [hunk 1]
```diff
@@ -149,7 +149,6 @@ domain:
         timeout:
```
Comment (AlbsAgren) [inline middleware/configuration/documentation/domain.general.operation.md:154]: Github wont let me comment on the right lines, but on 53 & 132 discoverable is listed as a property, but it's now called 'visibility', right?

**Review by AlbsAgren — DISMISSED** [review]

File: middleware/configuration/documentation/domain.general.operation.md (hunk 1 above)
Comment (lazan) [reply middleware/configuration/documentation/domain.general.operation.md:154]: correct, good catch!

Comment (lazan) [reply middleware/configuration/documentation/domain.general.operation.md:154]: I'll fix that. But you're ok with the rest?

Comment (AlbsAgren) [reply middleware/configuration/documentation/domain.general.operation.md:154]: Yeah, looks good to me

**Pull request merged**